microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,101 @@ These Release Notes provide information about new features and bug fixes that ar
 the template to copy for incoming sections is at end of this file
 
 this webpage covers the most recent ~5 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers: 
-Feb 2024
-Jan 2024
+Mar 2025
+Feb 2025
+Jan 2025
 [Dec 2024]
 Nov 2024
 Oct 2024
-Sep 2024
 
 moved to archive:
-Aug 2024
+Sep 2024
 -->
 
 
+<!-- Mar 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3124.44
+
+Release Date: March 10, 2025
+
+[NuGet package for WebView2 SDK 1.0.3124.44](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3124.44)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 134.0.3124.44 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No additional APIs have been promoted to Stable and added in this Release SDK.
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Disabled `kExtensionDisableUnsupportedDeveloper` in WebView2.  ([Issue #5113](https://github.com/MicrosoftEdge/WebView2Feedback/issues/5113))<!-- todo: correct spelling? not found in any docset, not mentioned in the Issue - more relevant wording for users?  where can confirm spelling? -->
```

Comment:
Where can one confirm correct spelling `kExtensionDisableUnsupportedDeveloper`?
Not found yet in any docset.
Not mentioned in the Issue https://github.com/MicrosoftEdge/WebView2Feedback/issues/5113 

Provide more relevant wording for users.
@sivMSFT @soumamazu

Reply:
```
* Extensions won't get disabled in WebView2 with the `AddBrowserExtensionAsync`, regardless of whether developer mode is on or off.  ([Issue #5113](https://github.com/MicrosoftEdge/WebView2Feedback/issues/5113))
```